backend/fms_core/template_importer/row_handlers/sample_submission/sample.py

```diff
@@ -27,8 +27,8 @@ def validate_row_input(self, **kwargs):
         # make sure all required values are present. WIP.
         if sample["sample_kind"] is None:
             self.errors['sample_kind'] = [f"Sample Kind is a required field."]
-        if sample["volume"] is None:
-            self.errors['volume'] = [f"Volume (uL) is a required field."]
+        if sample["volume"] is None and sample["volume_ratio"] is None:
+            self.errors['volume'] = [f"'Either Volume (uL)' or 'Ratio Library In Pool' is required field."]
```

Comment:
You should be able to suggest which one is required with the template content (you can pull the sample type for this purpose if you need).

Reply:
DONE